godtools/ViewControllers/MenuViewController.swift

```diff
@@ -0,0 +1,136 @@
+//
+//  MenuViewController.swift
+//  godtools
+//
+//  Created by Devserker on 4/24/17.
+//  Copyright © 2017 Cru. All rights reserved.
+//
+
+import UIKit
+
+class MenuViewController: BaseViewController, UITableViewDataSource, UITableViewDelegate {
+    
+    @IBOutlet weak var tableView: UITableView!
+    let menuCellIdentifier = "cellIdentifier"
+    let general = ["language_settings", "about", "help", "contact_us", "notifications", "preview_mode_translators_only"]
+    let share = ["share_god_tools", "share_a_story_with_us"]
+    let legal = ["terms_of_use", "privacy_policy"]
+    let header = ["menu_general", "menu_share", "menu_legal"]
+    
+    override var screenTitle: String {
+        get {
+            return "settings".localized
+        }
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.setupStyle()
+    }
+
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+    }
+    
+    // MARK: UI
+    
+    fileprivate func setupStyle() {
+        self.view.backgroundColor = .clear
+        self.tableView.backgroundColor = .gtGreyLight
+        self.registerCells()
+    }
+    
+    // MARK: - Navigation Buttons
+    
+    override func configureNavigationButtons() {
+        self.addEmptyLeftButton()
+        self.addDoneButton()
+    }
+    
+    // MARK: - UITableViewDataSource
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
```

Comment:
If we are going to keep the delegate and datasource methods in this file, then please declare them in extensions.

In this case I think it makes sense to leave them because the possibility of reuse is very small.